drivers/adc/Kconfig.stm32

```diff
@@ -3,10 +3,21 @@
 # Copyright (c) 2019 Intel Corporation
 # Copyright (c) 2019 Endre Karlson
 # Copyright (c) 2019 Song Qiang <songqiang1304521@gmail.com>
+# Copyright (c) 2021 Marius Scholtz, RIC Electronics
 # SPDX-License-Identifier: Apache-2.0
 
 config ADC_STM32
 	bool "STM32 ADC driver"
 	depends on SOC_FAMILY_STM32
 	help
 	  Enable the driver implementation for the stm32xx ADC
+
+if SOC_SERIES_STM32F2X || SOC_SERIES_STM32F4X || SOC_SERIES_STM32F7X
+
+config ADC_STM32_SHARED_IRQS
+	bool "STM32 ADC shared interrupts"
+	default y
+	depends on ADC_STM32
+	help
+	  Enable the use of shared interrupts for famalies that only have a single interrupt for all ADC's
+endif
```

Comment:
Please add a newline to the end of the file.

Reply:
The compliance check failed after adding the new line at the end of the file